cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApi.java

```diff
@@ -38,10 +38,7 @@ public KafkaConnectApiImpl(Vertx vertx) {
     }
 
     @Override
-    public Future<JsonObject> createOrUpdatePutRequest(
-            String host, int port,
-            String connectorName,
-                                                       JsonObject configJson) {
+    public Future<JsonObject> createOrUpdatePutRequest(String host, int port, String connectorName, JsonObject configJson) {
```

Comment:
I know it's a mess here, but it's not about our code. Let's leave this trivial PR for the main Strimzi repo. You can do that right now :-)